pkg/cluster/cluster.go

```diff
@@ -82,6 +83,8 @@ type Cluster struct {
 	currentProcess   Process
 	processMu        sync.RWMutex // protects the current operation for reporting, no need to hold the master mutex
 	specMu           sync.RWMutex // protects the spec for reporting, no need to hold the master mutex
+
+	backupJob        *batchv1beta1.CronJob  // periodical logical backups independent from WAL archiving
```

Comment:
Maybe we can be more specific in the variable name, `logicalBackupJob`? Just in case if there would be other kind of backups in the future.